packages/webos/VoiceControl/VoiceControl.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {findDOMNode} from 'react-dom';
+
+class VoiceControl extends React.Component {
```

Comment:
Architecturally, I have some concerns about this approach. It's HOC-like but not a HOC which makes it a bit out of place. I think I'd rather see it use `core/hoc`. Do we also need a solution for app-level event handling by listening to events on `document` rather than the node?

Reply:
I did change to HOC and it seems more appropriate structure. Thanks~
Additionally, in current status, web engine does not support event handling base on document object.(Because they are searching for information base on dom node) But I think app scenario can be covered using by node because app developer can create node to top level of application and it can be handle voice event globally.